Automattic/jetpack
# My Jetpack: Refresh product list after connecting

Fixes https://github.com/Automattic/jetpack-marketing/issues/972

## Proposed changes:

* add an endpoint to get product ownership information
* updating product list after connection status changes

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Create a JN site from this branch
* Go to `My Jetpack` and check `My products` cards
* Connect Jetpack
* Verify that `My products` cards update without reloading the page after the connection is established
